### DISCUSSION TITLE
Option for hiding new UI Panels

### DISCUSSION THREAD

**Discussion by AyrtonAlbuquerque** [Kong/insomnia#5557] [Feedback]
First, great job on the new release, it's looking really good. I would like to suggest adding an option to hide/toggle the new projects panel and and the new status/preferences bar to allow for a even more minimalistic view.

![Bloat](https://user-images.githubusercontent.com/36254780/208557526-c3a39881-5a2c-420e-ae07-fb80df7340d5.png)

**Comment by jmoniatte** [comment]
This would be awesome specially for "Insomnia Individual" subscribers for which "Teams" do not apply.

**Comment by DusanLesan** [comment]
I completely agree. I made another related https://github.com/Kong/insomnia/discussions/5546. I dislike having UI elements of no use to me especially if they are this big

**Comment by subnetmarco** [comment]
It is possible to now hide the organizations sidebar since Insomnia 9.x (or maybe even before that, it's been a while). The footer is functional in the application (that's where the preferences are, for example) but we may offer in the future the ability to also remove that.

You can press this button:

![Screenshot 2024-07-02 at 6 46 51 PM](https://github.com/Kong/insomnia/assets/964813/a7041e82-d6c7-4d37-92c1-d097dbc7c2a5)
